### DISCUSSION TITLE
model finetune over pre-trained model weights..

### DISCUSSION THREAD

**Discussion by tzktok** [PaddlePaddle/PaddleOCR#13180] [Q&A]
#### 问题描述 / Problem Description
i have the trained detection model...i want to fine tune some more dataset over that weight...how to do that..
```
tamil@DESKTOP-FLC1RG0:~/PaddleOCR/inference/default_model$ ls
inference.pdiparams  inference.pdiparams.info  inference.pdmodel
```
this are the above files in my default model folder..i want to a finetune above this model..how to achive that..
@GreatV  @ZeyuChen 

#### 运行环境 / Runtime Environment
- OS:Ubuntu
- Paddle:addlepaddle-2.6.1
- PaddleOCR:paddleocr 2.7.3

**Comment by GreatV** [comment]
please refer to: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/training_en.md

**Comment by tzktok** [comment]
my trainning comment...
```
tamil@DESKTOP-FLC1RG0:~/PaddleOCR$ python3 tools/train.py -c configs/det/det_mv3_db.yml -o Global.checkpoints=./en_PP-OCRv3_det_distill_train/best_accuracy
```
but pretrained weight file is  missing in downloaded folder...
```
[2024/06/25 15:34:04] ppocr WARNING: ./en_PP-OCRv3_det_distill_train/best_accuracy.pdopt is not exists, params of optimizer is not loaded
```
![Screenshot (16)](https://github.com/PaddlePaddle/PaddleOCR/assets/169971463/896c4bef-5b46-4ab4-98d6-33f807fd3ac1)
 i want to fine tune over this weight..when i downloaded the that trained model folder..it contains only..
![Screenshot (17)](https://github.com/PaddlePaddle/PaddleOCR/assets/169971463/91b207e5-ce90-4853-9efc-47fae65fb814)
  
.pdopt file is not availble ?  @GreatV 

how to fine tune en_PP-OCRv3_det ?.. what is correct config file for this finetune?
